js/client-chat.js

```diff
@@ -420,6 +420,43 @@
 				challenge(targets);
 				return false;
 
+			case 'accept':
+				if (toId(target)) return text;
+
+				var $challenges = $('.challenge').has('button[name="acceptChallenge"]');
+				if (!$challenges.length) {
+					app.addPopupMessage('You do not have any pending challenges to accept.');
+					this.parseCommand('/help accept');
+					return false;
+				}
+				if ($challenges.length > 1) {
+					app.addPopupMessage('You need to specify a user if you have more than one pending challenge to accept.');
+					this.parseCommand('/help accept');
+					return false;
+				}
+
+				var format = $('button[name="format"]', $challenges[0]).val();
+				if (window.BattleFormats[format].team === 'preset' || Object.keys(Storage.teams).length) return text;
```

Comment:
Simulating a click with both commands would also handle the HTML of the challenge box properly, which I hadn't caught earlier.